### PR TITLE
Add new filtersModalSize option for modal dialog && update docs

### PR DESCRIPTION
**Pull request by ksn135**
In modal dialog mode (`filtersMode: modal`) you can setup desirable modal dialog size with the specific 
keyword `filtersModalSize` (see committed pictures). Change the size of the modal dialog by specifing one of these values:
 * `medium` (default)
 * `large`
 * `small`

Example:
```yml
builders:
    list:
        params:
            title:              audit.title
            filtersMode:        modal
            filtersModalSize:   large # or small or medium (default)
            display: 
                - timestamp
                - user
                - ip
                - url
                - code
            sort: [timestamp, desc]
            actions: {}
            object_actions: {}
```

### CODE REVIEW

File: Resources/doc/builders/list/filters.md [hunk 1]
```diff
@@ -10,11 +10,19 @@
 By default, the current template provides two ways to display filters:
```
Comment (bobvandevijver) [inline Resources/doc/builders/list/filters.md]: This should be edited to three ;)

File: Resources/doc/builders/list/filters.md [hunk 2]
```diff
@@ -10,11 +10,19 @@
...
-keyword `filtersMode`. Two values are currently interpreted:
+keyword `filtersMode`. Three values are currently interpreted:
  * `top`: filters will be on top of results
- * 'default`: (default value), filters will be on the right of the results list
+ * `default`: (default value), filters will be on the right of the results list
+ * `modal`: filters will be displayed in modal dialog
+
+Also in modal dialog mode (`filtersMode: modal`) you can setup desirable modal dialog size with the specific 
+keyword `filtersModalSize`. Change the size of the modal by specifing one of these values:
+ * `medium` (default)
```
Comment (bobvandevijver) [inline Resources/doc/builders/list/filters.md]: I Would like the small, medium, large order, but that might be personal

**Comment by bobvandevijver** [comment]
@ksn135 Looks good, could you change the two things I commented?

**Comment by ksn135** [comment]
@bobvandevijver done

**Comment by bobvandevijver** [comment]
Merged, thanks!

**Pull request merged**